portal-impl/test/unit/com/liferay/portal/util/PortalImplUnitTest.java

```diff
@@ -148,10 +148,14 @@ public void testGetForwardedHostWithCustomXForwardedHostEnabled()
 		String webServerForwardedHostHeader =
 			PropsValues.WEB_SERVER_FORWARDED_HOST_HEADER;
 
+		String[] virtualHostsValidHosts = PropsValues.VIRTUAL_HOSTS_VALID_HOSTS;
+
 		try {
 			setPropsValuesValue("WEB_SERVER_FORWARDED_HOST_ENABLED", true);
 			setPropsValuesValue(
 				"WEB_SERVER_FORWARDED_HOST_HEADER", "X-Forwarded-Custom-Host");
+			setPropsValuesValue(
```

Comment:
For the failing test here, we would like to see when PropsValues.WEB_SERVER_FORWARDED_HOST_ENABLED is True then we get `forwardedHost` while calling `getForwardedHost()`; and when it is false, we will get serverName as a result. 
Therefore, set forwardedHost to be a valid virtual host should be a prerequisite in this test